test/external/atoms_calculators.jl

```diff
@@ -15,10 +17,52 @@
 
     energy = AtomsCalculators.potential_energy(silicon, calculator)
     ref_energy = -7.86059
-    @test energy ≈ ref_energy rtol=1e-3
+    @test austrip(energy) ≈ ref_energy rtol=1e-3
+end
+
+@testitem "Test AtomsCalculators energy forces interface" setup=[TestCases] begin
+    using DFTK
+    using AtomsBase
+    using AtomsCalculators
+    using Unitful
+    using UnitfulAtomic
+    silicon = TestCases.silicon
+
+    # Perturb from equilibrium so forces are not 0.
+    positions = [silicon.positions[1] + [0.05, 0, 0], silicon.positions[2]]
+    silicon = periodic_system(silicon.lattice, silicon.atoms, positions)
+
+    model_kwargs = (; temperature=1e-3, functionals=[:lda_x, :lda_c_pw])
+    basis_kwargs = (; kgrid=[4, 4, 4], Ecut=5.0)
+    scf_kwargs = (; tol=1e-7)
+    calculator = DFTKCalculator(; model_kwargs, basis_kwargs, scf_kwargs, verbose=true)
```

Comment:
This is now essentially 3 times the same code. Can you make a function singling this out (see other files how this is done).

Reply:
fixed